src/components/structures/LoggedInView.tsx

```diff
@@ -694,8 +695,10 @@ class LoggedInView extends React.Component<IProps, IState> {
                 >
                     <ToastContainer />
                     <div className={bodyClasses}>
-                        <div className='mx_LeftPanel_wrapper'>
-                            { SettingsStore.getValue('TagPanel.enableTagPanel') &&
+                        <div className='mx_LeftPanel_outerWrapper'>
```

Comment:
in the past we've had performance issues with adding layers to the left panel like this, but it's entirely possible that they have been fixed over the years. Would recommend watching for left panel-related performance complaints post-merge, particularly for large accounts and Chrome users.

Reply:
surprisingly, it wasn't a performance issue but rather a scrolling issue this time 😭 https://github.com/matrix-org/matrix-react-sdk/pull/8109